libraries/AP_Logger/LogStructure.h

```diff
@@ -1187,14 +1187,14 @@ struct PACKED log_PSCZ {
 // @Field: TPY: Target position relative to origin, Y-axis
 // @Field: PX: Position relative to origin, X-axis
 // @Field: PY: Position relative to origin, Y-axis
-// @Field: TVX: Target velocity, X-axis
-// @Field: TVY: Target velocity, Y-axis
-// @Field: VX: Velocity, X-axis
-// @Field: VY: Velocity, Y-axis
-// @Field: TAX: Target acceleration, X-axis
-// @Field: TAY: Target acceleration, Y-axis
-// @Field: AX: Acceleration, X-axis
-// @Field: AY: Acceleration, Y-axis
+// @Field: TVX: Target velocity, earth-frame X-axis
+// @Field: TVY: Target velocity, earth-frame Y-axis
+// @Field: VX: Velocity, earth-frame X-axis
+// @Field: VY: Velocity, earth-frame Y-axis
+// @Field: TAX: Target acceleration, body-frame X-axis
```

Comment:
Fairly sure there all earth frame. Might as well add North/East too.

Reply:
yes, "earth-frame X-axis" is really North.  earth-frame Y axis is East.  So it could be "TVX: Target velocity North".

I also think that the accelerations are all earth frame as well.

Reply:
So generated this graph with
```
mode loiter
arm throttle
rc 3 1600
rc 3 1500
rc 2 1000
```

![image](https://user-images.githubusercontent.com/7077857/126415599-f23a8846-3cc9-44da-8bca-48cccfd073cb.png)

As for North/South vs X/Y - I guess that's because we're not necessarily aligned with *actual* North/East depending on what the AHRS is saying - e.g. if you're using pure visual navigation you might not know where magnetic north is.  However, 99.999% of our users don't care about that, so I've pushed up a patch which changes them to N/E

Reply:
(the speeling error in that graph is almost certainly because @IamPete1 is involved in this PR)

Reply:
These are body frame x and y accelerations but they are rotated about the earth z axis to north and east.  it is essentially the accelerations in the body axis due to the lean angles in the north and east directions, if that makes sense :-)

Reply:
Yes, these should no have "body-frame" in the description.

Reply:
I've removed "body_frame".  Would be nice to wedge @bnsgeyer 's explanation in there somehow, but I couldn't see how to do that easily.  Perhaps it could go into the message description?